core/src/env/srml/srml_only/impls.rs

```diff
@@ -145,6 +147,21 @@ where
             .expect("we can expect to receive a correctly sized buffer here")
     }
 
+    fn now() -> <Self as EnvTypes>::Moment {
```

Comment:
Please change this to also use the new https://github.com/paritytech/ink/blob/master/core/src/env/srml/srml_only/impls.rs#L97 functionality.